securesystemslib/key.py

```diff
@@ -136,10 +142,10 @@ def from_securesystemslib_key(cls, key_dict: Dict[str, Any]) -> "SSlibKey":
             ) from e
 
         return cls(
-            key_dict["keyid"],
             key_meta["keytype"],
             key_meta["scheme"],
             key_meta["keyval"],
+            key_dict["keyid"],
```

Comment:
Is `keyid` optional for a securesystemslib generated dictionary?
I don't think so, but please do check as it will create `KeyError` in some cases if `keyid` is not populated.

Reply:
I don't think if there is any case of `keyid` is not populated. Because it's required as mentioned in `ANYKEY_SCHEMA` and all the generator functions.